palisade-service/mvn_dependency_tree.txt

```diff
@@ -136,13 +136,13 @@ io.etcd/jetcd-launcher/version = 0.3.0
 
 io.etcd/jetcd-resolver/version = 0.3.0
 
-io.fabric8/kubernetes-client/version = 4.3.0
+io.fabric8/kubernetes-client/version = 4.2.0
```

Comment:
Please fix downgraded dependency.

Reply:
Running mvn dependency:tree (-X) shows the following:
This is a dependency from the top level i.e a depency introduced by the parent:
```
    <parent>
        <artifactId>spring-boot-starter-parent</artifactId>
        <groupId>org.springframework.boot</groupId>
        <version>2.1.8.RELEASE</version>
```

Reply:
Probably just that they haven't updated that version in the time that version of Spring was released. We'll leave this for now, it should upgrade as part of moving to new Spring releases.